File: andrew035/Questao05.py
print("** Seguradora XYZ **")
print (">> Sistema de avaliação <<")

n = int(input("Qtd de contratos simples: "))
c = int(input("Qtd de contratos completos: "))

simples = n * 230
completo = c * 350
Lucro = simples + completo

print("Lucro obtido: " + str(Lucro))

File: andrew035/Questao05.java
import java.util.Scanner;

public class Questao05{
        public static void main(String[] args){
        Scanner s;
        s = new Scanner(System.in);

        System.out.println("**Seguradora XYZ**");
        System.out.println(">> Sistema de avaliação <<");

        System.out.print("Qtd de contratos simples: ");
        int n = s.nextInt();

        System.out.print("Qtd de contratos completos: ");
        int c = s.nextInt();

        int simples = n * 230;
        int completo = c * 350;
        int Lucro = simples + completo;

        System.out.println("Lucro obtido: " + Lucro);}}
 
